tasks/add_new_openwrap_partner.py

```diff
@@ -374,13 +384,55 @@ def get_dfp_targeting(self):
             if self.bid_price: 
                 top_set['children'].append(self.bid_price)
 
+            # For HVLI, set both pwtbb and pwtecp with OR condition
+            # Targeting keys will be formed like
+            # pwtecp IS (1.*) AND pwtbst=1 AND pwtpid = abc
+            # or
+            # pwtpb = 1.0 AND pwtbst=1 AND pwtpid = abc
+            if self.price_bucket:
+                #pwtecp
+                cust_key_set1 = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'AND',
+                    'children': []
+                }
+                cust_key_set1['children'].append(top_set)
+                cust_key_set1['children'].append(self.price_set)
+
+                #pwtpb
+                cust_key_set2 = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'AND',
+                    'children': []
+                }
+                cust_key_set2['children'].append(top_set)
+                cust_key_set2['children'].append(self.price_bucket)
+
+                #final keys will be OR of both sets
+                final_cust_keys = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'OR',
+                    'children': [cust_key_set1, cust_key_set2 ]
+                }
+                return final_cust_keys
+            else:
+                #Include only 'pwtecp' in the final targeting key set
+                #eg: pwtecp IS (1.*) AND pwtbst=1 AND pwtpid = abc
+                if self.price_set:
+                     top_set['children'].append(self.price_set)
+
         return top_set
 
     def process_price_bucket(self, start_index, end_index, granu):
 
         subCustomValueArray = []
         sub_granu = None
 
+        hvli = False
```

Comment:
Nitpick:  can we rename it to `highest_val_lineitem`,  ` hvli` is  bit confusing.